api/api/src/main/scala/uk/ac/wellcome/platform/api/Main.scala

```diff
@@ -43,7 +43,11 @@ object Main extends WellcomeTypesafeApp {
           .getOrElse("context.json"),
       )
 
-    val router = new Router(elasticClient, elasticConfig, apiConfig)
+    val queryConfig =
+      QueryConfig.fetchFromIndex(elasticClient, elasticConfig.imagesIndex)
```

Comment:
Would this need to be refreshed from the index every so often? :thinking:

Reply:
Good question - I think in this case no, because changing the parameters of the inferrer requires a complete reindex (as the vectors only make sense with respect to each other) and the hardcoded index means the API task is bound to the index it starts with